deployer/utils.py

```diff
@@ -88,6 +83,7 @@ def verify_and_decrypt_file(encrypted_path):
             decrypted contents. Unless the file is not valid JSON/YAML or does not have
             the prefix `enc-`, then we return the original, encrypted path.
     """
+    assert_file_exists(encrypted_path)
```

Comment:
The functions parameter is named encrypted_path and has a docstring that doesn't indicate it would accept a non-encrypted file.

I think it makes sense to update the function name/parameter name in line with `get_decrypted_files` to for example `get_decrypted_file` and clarify that it will: a) return a file path as it is if its not encrypted or b) return a temporary decrypted file if it was encrypted, and c) run verifications that the file exists and wasn't already decrypted if it had certain naming conventions to indicate it should be encrypted.